src/OpenLoco/Windows/News/News.cpp

```diff
@@ -328,12 +328,8 @@ namespace OpenLoco::Ui::Windows::NewsWindow
             {
                 _dword_525CD0 = ecx;
                 _dword_525CD4 = edx;
-                auto viewport = self->viewports[0];
-                if (viewport != nullptr)
-                {
-                    viewport = nullptr;
-                    self->invalidate();
-                }
+                self->viewportRemove(0);
+                self->invalidate();
```

Comment:
invalidate is a relatively light function to call so may as well take it out of the if.